webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
```

Comment:
I found out it's much easier to maintain if you separate your formaters into `component` and `formater` who only render the relevant component.
See:
https://github.com/Katello/katello/tree/master/webpack/move_to_foreman/components/common/table